pulp_smash/utils.py

```diff
@@ -86,6 +86,36 @@ def os_is_f27(cfg, pulp_host=None):
     return response.returncode == 0
 
 
+def fips_is_supported(cfg, pulp_host=None):
+    """Return ``True`` if the server has crypto fips option available or ``False`` otherwise
+    :param cfg: Information about the system being targeted
+    :param pulp_host: A :class : `pulp_smash.config.PulpHost` to target, instead of the default chosen by :class: pulp_smash.cli.Client`.
+    :return: True of False
+    """
+    try:
+       cli.Client(cfg, cli.echo_handler, pulp_host).run((
```

Comment:
Why use `cli.echo_handler`? This callback always returns the responses it processes without checking for non-zero return codes.